app/scripts/modules/core/src/pipeline/config/triggers/MetadataPageContent.tsx

```diff
@@ -104,33 +102,30 @@ export function MetadataPageContent(props: IMetadataPageContentProps) {
   const { pipeline, updatePipelineConfig } = props;
 
   return (
-    <SpinFormik<Partial<IPipeline>>
-      initialValues={pipeline}
-      onSubmit={() => {}}
-      render={(formik) => (
-        <>
-          <WatchValue onChange={updatePipelineConfig} value={formik.values} />
```

Comment:
This SpinFormik made a copy of the pipline as internal state from the `initialValues` prop.  Then, any time the description or tags were edited, it clobbered all pipeline fields with the original values that it initially saved.

I switched this to a controlled FormField to avoid confusion.

Reply:
Oh geez, that's right. 🤦

Reply:
Thanks for fixing this